### PR TITLE
Make order of generated yaml from kubebuilder create config --crds deterministic

### PR DESCRIPTION
**Pull request by pmorie**
... to facilitate adding test cases for validations etc in different CRDs.

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -200,6 +200,121 @@ EOF
...
+        status:
+          type: object
+      type: object
+  version: v1beta1
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: null
+EOF
```
Comment (droot) [inline test.sh]: I took another look and realized that `deterministic ordering` can be unit tested in Go itself. This feels like an anti-pattern to have that functionality tested here `test.sh` ?

Comment (pmorie) [reply test.sh]: Yeah - i’ll move the test to go

**Comment by pwittrock** [comment]
@pmorie I am also fine with making that a TODO and not blocking this on that.

**Comment by pwittrock** [comment]
Since the point of this PR is to increase test coverage, I am going to merge it.  We can move logic from the test.sh to e2e tests in go independent of this PR.

**Pull request merged**